src/Components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,214 @@
+/* eslint-disable @typescript-eslint/no-shadow */
```

Comment:
it's not ok

Reply:
It was a temporary decision just for don't see linter errors)